webapp/src/components/markdownEditorInput/markdownEditorInput.tsx

```diff
@@ -38,9 +47,38 @@ type Props = {
 const MarkdownEditorInput = (props: Props): ReactElement => {
     const {onChange, onFocus, onBlur, initialText, id, isEditing} = props
     const boardUsers = useAppSelector<IUser[]>(getBoardUsersList)
-    const mentions: MentionData[] = useMemo(() => boardUsers.map((user) => ({name: user.username, avatar: `${imageURLForUser ? imageURLForUser(user.id) : ''}`, is_bot: user.is_bot})), [boardUsers])
+    const board = useAppSelector(getCurrentBoard)
     const ref = useRef<Editor>(null)
 
+    const [suggestions, setSuggestions] = useState<Array<MentionUser>>([])
+
+    const loadSuggestions = async (term: string) => {
+        let users: Array<IUser>
+
+        if (board && board.type === BoardTypeOpen) {
+            users = await octoClient.searchTeamUsers(term)
+        } else {
+            users = boardUsers
+        }
+
+        const mentions = users.map(
+            (user) => ({
+                name: user.username,
+                avatar: `${imageURLForUser ? imageURLForUser(user.id) : ''}`,
+                is_bot: user.is_bot}
+            ))
+        setSuggestions(mentions)
+    }
+
+    const debouncedLoadSuggestion = useMemo(() => debounce(loadSuggestions, 200), [])
+
+    useEffect(() => {
+        // Get the ball rolling. Searching for empty string
+        // returns first 10 users in alphabetical order.
+        debouncedLoadSuggestion('')
```

Comment:
We could use the `loadSuggestions` directly on the initial request

Reply:
Done!